Activity.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { Text, View, Button, Alert, Vibration } from "react-native";
```

Comment:
you don't need to import the Text, View and the Button components since you're creating styled components, that library will sort this out itself.